src/main/java/com/compstore/dto/pc/PCFilteringResponseDTO.java

```diff
@@ -0,0 +1,15 @@
+package com.compstore.dto.pc;
+
+import com.compstore.dto.PagingAndSortingMetadataDTO;
+import java.util.List;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
```

Comment:
Dodany builder dzięki czemu przyjemniej tworzy się obiekt. Widać to metodzie serwisu `searchPC` jak to działa